client/my-sites/subscribers/mutations/use-subscriber-remove-mutation.ts

```diff
@@ -83,14 +83,15 @@ const useSubscriberRemoveMutation = (
 					await Promise.all( promises );
 				}
 
-				let wasRemoved = false;
+				let emailRemoved = false;
+				let wpcomRemoved = false;
 
 				// Remove the subscriber from the followers if they have a numeric user_id
 				const numericUserId = Number( subscriber.user_id );
 				if ( ! isNaN( numericUserId ) ) {
 					try {
 						await wpcom.req.post( `/sites/${ siteId }/followers/${ numericUserId }/delete` );
-						wasRemoved = true;
+						wpcomRemoved = true;
 					} catch ( e ) {
 						// Only throw if they don't have an email subscription ID to try next
 						const emailSubscriptionId = getEmailSubscriptionId( subscriber );
```

Comment:
This predates this PR, but as written it sometimes runs twice. I'm not sure how costly that call is, but if we pulled it out of here and ran it before the `if`, we could remove the second `const emailSubscriptionId` declaration.

Reply:
Thanks for catching that, removed in: https://github.com/Automattic/wp-calypso/pull/103306/commits/d774a8fb6787f89bef574ce1adb73566b7c3cf03